### ISSUE TITLE
New kubeVersion requirement blocks deployments on EKS

### ISSUE THREAD

**Issue by mogggggg** [hashicorp/consul-helm#872]
Hey there,
There was a [recent PR](https://github.com/hashicorp/consul-helm/pull/870) adding a `kubeVersion` requirement to the Helm chart - unfortunately this stops deployment on EKS.

I'm using the latest version of the chart (`1.31.0`) and the latest version of EKS (`1.19`) but when I try to deploy I get the following error:
```console
Error: chart requires kubeVersion: >= 1.13.0 which is incompatible with Kubernetes v1.19.6-eks-49a6c0
```

**Comment by david-yu** [comment]
@mogggggg Thanks for the heads up. We'll look into this tomorrow, its interesting that it fails on EKS K8s 1.19 when that is greater than 1.13. You should be able to just remove that line from Chart.yaml for now.

**Comment by mogggggg** [comment]
Thanks @david-yu! Yup, it's not a big deal for us, we're using `0.30.0` instead.

I haven't done any testing but I assume it's because AWS suffix the version number with their own info - let me know if you want me to check anything.

**Comment by david-yu** [comment]
No problem, we definitely want you to try out the latest and greatest without having to manually modify the chart. We should have a release that reverts this tomorrow! Sorry for the inconvenience.

**Comment by tbuchier** [comment]
Hello, same error here, we were waiting for the v0.31.0 for the metrics support, so I guess i'll override the Chart.yaml until it's fixed.

**Comment by ishustava** [comment]
Hey all! Sorry about that! 

We've just released a patch release [v0.31.1](https://github.com/hashicorp/consul-helm/releases/tag/v0.31.1) that should fix this issue. 

Let us know if you still see problems!

**Comment by tbuchier** [comment]
Working well now, thanks for the quick fix :)
